xml/art_admin_slemicro_selinux.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+                 type="text/xml"
+                 title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<article xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="article-selinux" xml:lang="en">
+ <title>&slema; security</title>
+ <section xml:id="sec-introduction-selinux">
+  <title>Introduction to SELinux</title>
+  <para>
+   SELinux was developed as an additional Linux security solution that uses the
+   security framework in the Linux kernel. The purpose was to allow for a more
+   granular security policy that goes beyond the standard Discretionary Access
+   Controls (DAC), the traditional file permissions of owner/group/world, and
+   read/write/execute.
+  </para>
+  <para>
+   SELinux uses labels attached to objects (for example, files, binaries, and
+   network sockets) and uses them to determine privilege boundaries, thereby
+   building up a level of confinement that can span more than a process, or
+   even the whole system.
+  </para>
+  <para>
+   SELinux was developed in accordance with some certification standards like
+   Common Criteria and FIPS 140.
+  </para>
+  <para>
+   SELinux default action is to deny any access that would violate rules
+   defined by the policy. Other feature of SELinux that increases security is
+   that processes run within their own domain and cannot access files of other
+   processes that run in different domains. A process cannot access another
+   process as well, unless configured otherwise.
+  </para>
+  <para>
+   SELinux was designed to enhance existing security solutions not to replace
+   them, for example, discretionary access control is still applied even
+   if the system is using SELinux. If DAC denies access first, SELinux policies
+   are not applied.
```

Comment:
rephrase this to emphasize that SELinux doesn't get involved. The policy is still applied, it's just not used in this moment